packages/containers/src/SidePanel/SidePanel.connect.js

```diff
@@ -72,8 +72,11 @@ function getActionsWrapped(actions) {
 	});
 }
 
-function getSelectedAction(currentRoute, actions) {
-	return actions.find(action => action.href && isBasePathOf(action.href, currentRoute));
+function getSelectedAction(currentRoute, actions, basename) {
+	const getFullPath = href => (basename ? `${basename}${href}`.replaceAll('//', '/') : href);
+	return actions.find(
+		action => action.href && isBasePathOf(getFullPath(action.href), currentRoute),
```

Comment:
Build the full path (router config + route basename) before searching for the current menu.